### PR TITLE
Stepup locale cookie

### PR DESCRIPTION
**Pull request by relaxnow**
@rjkip review?

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -173,6 +173,13 @@ surfnet_stepup:
         originator: %sms_originator%
         otp_expiry_interval: %sms_otp_expiry_interval%
         maximum_otp_requests: %sms_maximum_otp_requests%
+    locale_cookie:
+        name: stepup_locale
+        domain: %locale_cookie_domain%
+        expire: 0
+        path: /
+        secure: true
+        httpOnly: false
```
Comment (rjkip) [inline app/config/config.yml]: Any reason to not enable this?

File: src/Surfnet/StepupGateway/GatewayBundle/Saml/Proxy/ProxyStateHandler.php [hunk 2]
```diff
@@ -261,6 +261,24 @@ public function getResponseContextServiceId()
...
+     * @return $this
+     */
+    public function setPreferredLocale($locale)
+    {
+        $this->set('locale', $locale);
+        return $this;
+    }
+
+    /**
+     * @return string
```
Comment (rjkip) [inline src/Surfnet/StepupGateway/GatewayBundle/Saml/Proxy/ProxyStateHandler.php]: This can be null.

**Comment by rjkip** [comment]
Nice and simple integration :+1:

File: app/config/config.yml (hunk 1 above)
Comment (relaxnow) [inline app/config/config.yml]: There are no secrets in it, seems useful to let client side code access the value.

Comment (rjkip) [inline app/config/config.yml]: But then I must ask: which client-side code currently needs this? :no_mouth:

**Comment by relaxnow** [comment]
@rjkip rereview?

**Comment by rjkip** [comment]
:ship:

**Comment by rjkip** [comment]
Pending build status, :+1::ship:

**Pull request merged**